### ISSUE TITLE
New release on RubyGems

### ISSUE THREAD

**Issue by pedro-stanaka** [refile/refile#574]
I needed to use refile with Rails 5 which requires rack 2.5. But version 0.6.2 of refile uses `sinatra ~> 1.4.6`, so we need a new release of refile in ruby gems to upgrade sinatra to newer version.

**Comment by optiontech** [comment]
Any word on a new gem for Rails 5?  I'm currently using the master branch in my Gem file but I'd like to lock down the version for production so I don't get unanticipated changes.  Thanks so much!

**Comment by wdiechmann** [comment]
I forked master but 

1) if I run `rspec` right of the bat with ruby 2.5.1p57 and rails 5.2.1 and rack 2.0.5 in the Gemfile I foreseeably get:

```ruby

An error occurred while loading ./spec/refile/attachment/active_record_spec.rb.
Failure/Error:
  class TestMigration < ActiveRecord::Migration
    def self.up
      create_table :posts, force: true do |t|
        t.integer :user_id
        t.column :title, :string
        t.column :image_id, :string
        t.column :document_id, :string
        t.column :document_filename, :string
        t.column :document_content_type, :string
        t.column :document_size, :integer

StandardError:
  Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migrationwas written for:

    class TestMigration < ActiveRecord::Migration[4.2]

```

2) fixing that error which aided by the error message is manageable does however point to an error which I am not so sure to having introduced with the 'upgrade' of rack!

```ruby

  4) Direct HTTP post file uploads Fail to upload a file that is too large
     Failure/Error: render text: "too large", status: 413

     ActionView::MissingTemplate:
       Missing template presigned_posts/upload, application/upload with {:locale=>[:en], :formats=>[:html, :text, :js, :css, :ics, :csv, :vcf, :vtt, :png, :jpeg, :gif, :bmp, :tiff, :svg, :mpeg, :mp3, :ogg, :m4a, :webm, :mp4, :otf, :ttf, :woff, :woff2, :xml, :rss, :atom, :yaml, :multipart_form, :url_encoded_form, :json, :pdf, :zip, :gzip], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby]}. Searched in:
         * "/Users/walther/Projects/frameworks/Gems/refile/spec/refile/test_app/app/views"
     # ./spec/refile/test_app/app/controllers/presigned_posts_controller.rb:24:in `upload'
     # ------------------
     # --- Caused by: ---
     # Capybara::ExpectationNotMet:
     #   expected to find text "Upload failure too large" in "Refile Test App\nTitle  \nDocuments files  \nDocument \nPresign start\nPresign complete\nPresign success\nUpload started\nUpload progress 3331 3331\nUpload failure Puma
```

3) which was a minor fix as well (adding an empty file `presigned_posts/upload.html.erb`) but that in turn pointed to

```ruby

  1) Multiple file uploads with presigned upload Successfully upload a file
     Failure/Error: expect(page).to have_content("Upload complete token accepted")
       expected to find text "Upload complete token accepted" in "Refile Test App\nTitle  \nDocuments files  \nDocument  \nPresign start\nPresign start\nPresign complete\nPresign success\nUpload started\nUpload progress 436 436\nPresign complete\nPresign success\nUpload started\nUpload progress 436 436\nUpload success <!doctype html> <html> <head> <script src=\"/assets/application-7261dbd02107cd5114f6c051db2a12a260541d1cd8826a28aa5b35615c0c9939.js\"></script></head> <body> <h1>Refile Test App</h1> </body> </html>\nUpload complete <!doctype html> <html> <head> <script src=\"/assets/application-7261dbd02107cd5114f6c051db2a12a260541d1cd8826a28aa5b35615c0c9939.js\"></script> </head> <body><h1>Refile Test App</h1> </body> </html>\nUpload success <!doctype html> <html> <head> <script src=\"/assets/application-7261dbd02107cd5114f6c051db2a12a260541d1cd8826a28aa5b35615c0c9939.js\"></script> </head> <body> <h1>Refile TestApp</h1> </body> </html>\nUpload complete <!doctype html> <html> <head> <script src=\"/assets/application-7261dbd02107cd5114f6c051db2a12a260541d1cd8826a28aa5b35615c0c9939.js\"></script> </head> <body> <h1>Refile Test App</h1> </body> </html>\nAll uploads complete"
     # ./spec/refile/features/multiple_upload_spec.rb:101:in `block (3 levels) in <top (required)>'

```
and a few others mentioning "Upload failure too large" missing from the response - and I cannot find where these responses are to be generated (no mentioning of the texts in either the controller, the refile.js or otherwise, except for the _spec files :(

To summarize: I got stalled on master not parsing the spec as is :(

**Comment by sobrinho** [comment]
@wdiechmann you have to fix your migration, the error is related to that and not refile itself.

Also, the refile spec is green on master.

<img width="1392" alt="screen shot 2018-12-27 at 13 20 11" src="https://user-images.githubusercontent.com/26460/50484939-3b10fb80-09da-11e9-9be8-94d1316111cc.png">

**Comment by sobrinho** [comment]
@optiontech you can fix the commit in the Gemfile:

```ruby
gem 'refile', github: 'refile/refile', ref: '4e69ef3a324387cb3af015bdd103618e2f112c90'
```

**Comment by simi** [comment]
@sobrinho there's still no release. Any news on this?

**Comment by jtmkrueger** [comment]
I'm still seeing this at 0.6.2 on rubygems.
